docs/sources/architecture/distributor.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Distributor
+weight: 1000
+---
+# Distributor Component
+
+This document builds upon the information in the [Loki Architecture](./) page.
+
+## Where does it live?
+
+The distributor is the first component on Loki's write path. It's responsible for validating, preprocessing, and applying a subset of rate limiting to incoming data before sending it to the ingester component. 
+
+## What does it do?
+
+### Validation
+
+The first step the distributor takes is to ensure that all incoming data is according to specification. This includes things like checking that the labels are valid Prometheus labels as well as ensuring the timestamps aren't too old or too new or the log lines aren't too long.
+
+### Preprocessing
+
+Currently the only way the distributor mutates incoming data is by normalizing labels. What this means is making `{foo="bar", bazz="buzz"}` equivalent to `{bazz="buzz", foo="bar"}`, or in other words, ensuring that the order of labels doesn't matter. This allows Loki to cache and hash them deterministically.
+
+### Rate limiting
+
+The distributor can also rate limit incoming logs based on the maximum per-tenant bitrate. It does this by checking a per tenant limit and dividing it by the current number of distributors. This allows the rate limit to be specified per tenant at the cluster level and enables us to scale the distributors up or down and have the per-distributor limit adjust accordingly. For instance, say we have 10 distributors and tenant A has a 10MB rate limit. Each distributor will allow up to 1MB/second before limiting. Now, say another large tenant joins the cluster and we need to spin up 10 more distributors. The now 20 distributors will adjust their rate limits for tenant A to `(10MB / 20 distributors) = 500KB/s`! This is how global limits allow much simpler and safer operation of the Loki cluster.
+
+**Note: The distributor uses the `ring` component under the hood to register itself amongst it's peers and get the total number of active distributors**
```

Comment:
We should clarify that it actually has it's own ring config which you have to specify for this to work at all, there is a ring config section in the distributors which has to be populated for this global rate limiting to work. it's an entirely separate entry from the ingesters ring settings